llvm/lib/Transforms/Instrumentation/LowerAllowCheckPass.cpp

```diff
@@ -107,7 +108,18 @@ static bool removeUbsanTraps(Function &F, const BlockFrequencyInfo &BFI,
       case Intrinsic::allow_runtime_check: {
         ++NumChecksTotal;
 
-        bool ToRemove = ShouldRemove(BB);
+        bool ToRemove = ShouldRemoveRandom();
+
+        unsigned int cutoff = 0;
```

Comment:
lambda for `GetCuttoff(II)`

and please keep `ShouldRemove`, now as ShouldRemove(II)

Reply:
Done. I ended up using both BB and II:
```
  auto ShouldRemove = [&](const BasicBlock &BB, const IntrinsicInst *II) {
    unsigned int cutoff = GetCutoff(II);
    return ShouldRemoveRandom() || ShouldRemoveHot(BB, cutoff);
  };
```